42/README.md

```diff
@@ -0,0 +1,71 @@
+```
+shortname: [42/HARD-FORKS]
+name: Handling Hard Forks
```

Comment:
I would call it `Handling Forks` because the `Type 1` approach below can be used to govern "Soft Forks" (backwards-compatible validation logic updates) too.